x-pack/plugins/cases/public/components/create/form_context.tsx

```diff
@@ -62,6 +63,9 @@ export const FormContext: React.FC<Props> = ({
       isValid
     ) => {
       if (isValid) {
+        const selectedOwner = dataWithoutConnectorId.selectedOwner;
+        delete dataWithoutConnectorId.selectedOwner;
```

Comment:
Instead of mutating the passed in parameter how about we spread `dataWithoutConnectorId` and remove `selectedOwner` there?

Something like this:

```
const { selectedOwner, ...userFormData } = dataWithoutConnectorId
```

Reply:
sorted :D